src/@next/InlineError/InlineError.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { StyledText, StyledWrapper } from './InlineErrorStyle';
+import { Icon } from '../Icon';
+import { Red } from '../utilities/colors';
+
+export interface InlineErrorProps extends React.HTMLAttributes<HTMLDivElement> {
+  text: string;
+}
+
+export const InlineError = ({ text, ...props }: InlineErrorProps) => {
+  const error = !!text;
+
+  return error ? (
+    <StyledWrapper {...props}>
+      <Icon
+        name="ri-error-warning-fill"
+        fill={Red.B93}
+        width={20}
+        height={20}
+      />
+      <StyledText variant="subtitle2" color={Red.B93}>
```

Comment:
Maybe can set the color in `StyledText`? 🤔

Reply:
Updated